docs/apache-airflow/tutorial.rst

```diff
@@ -376,3 +376,165 @@ Here's a few things you might want to do next:
       - Review the :ref:`List of operators <pythonapi:operators>`
       - Review the :ref:`Macros reference<macros>`
     - Write your first pipeline!
+
```

Comment:
"What's next" should be the last section on this page. This section directs the user to other pages, so many users won't read this article further.